rt_gene/src/rt_gene/gaze_estimation_models_pytorch.py

```diff
@@ -211,82 +184,60 @@ def __init__(self, num_out=2):
         for param in self.right_features.parameters():
             param.requires_grad = True
 
-        _num_ftrs = _left_model.fc.in_features + _right_model.fc.in_features + 2  # left, right and head_pose
-        self.classifier = GazeEstimationAbstractModel._create_fc_layers(num_out=num_out, num_features=_num_ftrs)
+        self.xl, self.xr, self.concat, self.fc = GazeEstimationAbstractModel._create_fc_layers(in_features=_left_model.fc.in_features, out_features=num_out)
         GazeEstimationAbstractModel._init_weights(self.modules())
 
     def forward(self, left_eye, right_eye, headpose):
         left_x = self.left_features(left_eye)
         left_x = torch.flatten(left_x, 1)
+        left_x = self.xl(left_x)
```

Comment:
It looks as if the `forward()` method can be moved in the `GazeEstimationAbstractModel`, it does exactly the same stuff in all subclasses.

Reply:
You'd be correct if we didn't include MobilenetV2; the pooling bit in MobilenetV2 is `left_x = left_x.mean([2, 3])` rather than `left_x = torch.flatten(left_x, 1)` in all the others...
Could abstract the pooling function as some lambda with a partial in the abstract method but seemed to be less readible that way.
Of course the whole thing would go away if we remove MobilenetV2....

Reply:
See above.